devolving_music/lib/Song_Scores.py

```diff
@@ -0,0 +1,32 @@
+from devolving_music.lib.elo_scoring import *
+class Song_Scores():
+    
+    def __init__(self,Song):
```

Comment:
```suggestion
    def __init__(self, submission: SongSubmission):
```

1) Make sure there's a space between function arguments (this is also something pylint will catch)
2) Function arguments should be snake case, so not capitalized
3) The argument is a song submission, not a song, so the name should clarify that.
3) Please add typehints to function arguments when possible.

Reply:
![image](https://user-images.githubusercontent.com/22030302/174702101-ebe47577-83b8-4ccc-be9e-e43517fe71ac.png)
pylint doesn't catch the spaces between function arguments